manager-dashboard/app/components/GeoJsonFileInput/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import booleanValid from '@turf/boolean-valid';
+
+import FileInput, { Props as FileInputProps } from '#components/FileInput';
+import useMountedRef from '#hooks/useMountedRef';
+import GeoJsonPreview from '#components/GeoJsonPreview';
+
+function validateGeoJSON(value: GeoJSON.GeoJSON): boolean {
```

Comment:
In the existing manager dashboard we check how many features the geojson file contains and also check for the size of the area of interest depending on the zoom level defined by the project manager (not only the size of the file). Maybe I've missed this part, could you clarify if you implemented these checks as well?

Reply:
Yes, we have implemented the check.
We have a validation function defined here https://github.com/mapswipe/python-mapswipe-workers/blob/0d013617a60b747643ae21bd9b42fbff295fd687/manager-dashboard/app/views/NewProject/utils.ts#L104-L167